astropy/table/tests/test_column.py

```diff
@@ -796,7 +796,7 @@ def test_masked_col_unicode_sandwich():
     assert c[:].dtype.char == 'S'
 
     ok = c == ['abc', 'def']
-    assert ok[0] == True  # noqa
+    assert ok[0]
```

Comment:
@taldcroft , is it okay to remove `== True` here?

Reply:
Yes.